checkov/common/util/dockerfile.py

```diff
@@ -1,7 +1,7 @@
 import re
 
-DOCKER_FILE_MASK = re.compile(r"^(?:.+\.)?[Dd]ockerfile(?:\..+)?$")
+DOCKERFILE_MASK = re.compile(r"^(?:.+\.)?[Dd]ockerfile(?:\..+)?$(?<!\.[Dd]ockerignore)")
 
 
-def is_docker_file(file: str) -> bool:
-    return re.fullmatch(DOCKER_FILE_MASK, file) is not None
```

Comment:
can you leave `is_docker_file` and just call the new function from it + adding a deprecation warning
```python
warnings.warn("Please use is_dockerfile()", DeprecationWarning)
```
the other changes are all fine 🙂